instance/models/mixins/openedx_theme.py

```diff
@@ -55,29 +55,40 @@ class OpenEdXThemeMixin(models.Model):
     class Meta:
         abstract = True
 
-    def get_theme_settings(self):
+    def get_simple_theme_static_files_settings(self, application):
         """
-        Returns a text string with ansible variables for design fields (colors, logo, ...)
-        in YAML format, to be passed to configuration_theme_settings.
+        Returns a dict with the keys corresponding to the ansible variables supported by the 'simple-theme' role and
+        the values corresponding to the supported values for those variables.
         """
+        return [
+            {
+                "url": application.logo.url,
+                "dest": "lms/static/images/logo.png",
+            },
+            {
+                "url": application.favicon.url,
+                "dest": "lms/static/images/favicon.ico",
+            },
+        ]
 
-        if not self.deploy_simpletheme:
-            # This is the case e.g. of instances registered before the theme fields were available.
-            # We don't change their colors and we don't use simple_theme
-            return ""
-
-        # application can be None (for instances not created through the form) or a
-        # BetaTestApplication object. first() returns None or object.
-        application = self.betatestapplication_set.first()
-
-        if not application:
-            # Instance wasn't created from application form, so no colors will be set or changed
-            # and simple_theme won't be used (unless manually requested through other settings).
-            return ""
+    def get_common_settings(self, application):
+        """
+        Returns a dict with the simple-theme settings common to the v0 and v1 implementation.
+        """
+        return {
+            "SIMPLETHEME_ENABLE_DEPLOY": True,
+            "EDXAPP_COMPREHENSIVE_THEME_SOURCE_REPO": settings.SIMPLE_THEME_SKELETON_THEME_REPO,
+            "EDXAPP_COMPREHENSIVE_THEME_VERSION": settings.SIMPLE_THEME_SKELETON_THEME_VERSION,
```

Comment:
@lgp171188 Is there a way we can keep supporting the old theme for a while? 
We can add a temporary `SIMPLE_THEME_SKELETON_THEME_LEGACY_VERSION` for a while.

I want to deploy this production without breaking the current production deployments.